version/version.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2018, Google, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package version
+
+import (
+	"fmt"
+)
+
+// The git commit that was compiled. This is filled in by the compiler.
+var GitCommit string
+
+// Main version number being run right now.
+var Version = "0.4.0"
```

Comment:
Why is version information hardcoded in the source?
It is usually imposed from the outside. If the compiler is filling in the GitCommit,
why cant you also fill in the version (and release phase below)?

Reply:
We'd set it during a build/release cycle: spinnaker/spinnaker#3182 (look for `ldflags`). It's set here also so we have a reasonable user agent and version for built versions that haven't been released yet.